src/layouts/Resources.jsx

```diff
@@ -255,7 +255,10 @@ export default class Resources extends Component {
     return (
       <>
         {/* Resource Room exists */}
-        <Header />
+        <Header
+          backButtonText="Back to My Workspace"
+          backButtonUrl={`/sites/${siteName}/workspace`}
+        />
```

Comment:
Should Resources be redirecting to Workspace, since all other items in the nav bar redirect to Sites?

Reply:
fixed in [bf2adb0](https://github.com/isomerpages/isomercms-frontend/pull/206/commits/bf2adb0d48e4dcb163b47abdb186a7b720fc75d7)

Reply:
fixed in [7ddfd3](https://github.com/isomerpages/isomercms-frontend/pull/206/commits/7ddfd3a49c5f3a43cc378819625b54c356e025f6)